packages/meditrak-app/app/assessment/DumbSurveyScreen.js

```diff
@@ -28,7 +28,7 @@ const LENGTH_OF_TRANSITION = 300;
 export class DumbSurveyScreen extends React.Component {
   static navigationOptions = ({ navigation }) => ({
     headerTitle: navigation.state.params.headerLabel,
-    headerLeft: () => <HeaderLeftButton source={require('../images/x.png')} />,
+    headerLeft: () => <HeaderLeftButton source={require('../images/x.png')} labelVisible={false} />,
```

Comment:
I'm curious why we didn't need to put this for the default back buttons (ones that we don't have to replace the back image)...

Reply:
Also curious, but don't know!